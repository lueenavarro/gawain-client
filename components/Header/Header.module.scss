@use "styles/colors";
@use "styles/mixins";

.header {
  height: 50px;
  background-color: colors.$primary;
  color: colors.$white;

  &__container {
    @include mixins.container;
    display: flex;
    align-items: center;
    height: 100%;
  }

  &__title {
    font-weight: bold;
    cursor: pointer;
    user-select: none;

    &:hover {
      color: gray;
    }
  }


}

.account {
  margin-left: auto;
  display: flex;
  &__email {
    margin: 0;
    &::after {
      content: "|";
      margin: 0 10px;
    }
  }

  &__logout {
    cursor: pointer;
  }
}