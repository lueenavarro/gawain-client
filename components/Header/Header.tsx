import Link from "next/link";
import React from "react";

import styles from "./Header.module.scss";

const Header = () => {
  return (
    <header className={styles.header}>
      <div className={styles["header__container"]}>
        <Link href="/">
          <div className={styles["header__title"]}>ONESTEP</div>
        </Link>
      </div>
    </header>
  );
};

export default Header;
