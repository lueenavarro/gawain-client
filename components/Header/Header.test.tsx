import ReactDOM from "react-dom";
import renderer from "react-test-renderer";

import Header from "./Header";

test("render without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Header />, div);
});

test("snapshot matches", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
