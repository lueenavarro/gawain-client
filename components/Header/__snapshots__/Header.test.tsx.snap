// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot matches 1`] = `
<header
  className="header"
>
  <div
    className="header__container"
  >
    <div
      className="header__title"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      ONESTEP
    </div>
  </div>
</header>
`;
